content/docs/features.md

```diff
@@ -95,38 +96,43 @@ app_running
 
 ```
 
-### Online
+### Online generation
 
 <a download="ShoppingCart.graphml" href="/content/resources/ShoppingCart.graphml"><img src="/images/amazonShoppingCart_small.png" alt="Model" align="right"/></a>
 
-The path generation is done during the execution of the test, run-time. This means that GraphWalker needs to be embedded in your test automation code. This adds a bit of complexity, but the advantages are:
+The path generation is done during the execution of the test, run-time. 
+This means that GraphWalker needs to be embedded in your test automation code. 
+This adds a bit of complexity, but the advantages are:
 
 * No need to handle the intermediate path sequences in files.<br>{: style=""}
 
   If your test automation is written in Java, it's pretty straight forward. You annotate a Java class with `@GraphWalker`.<br>
-  The class represents you graph, and the methods in that class represents the elements in that graph. Using [maven](https://maven.apache.org/), running the test with GraphWalker is done by:
-  {: style="color:gray; font-size: 90%"}
+  The class represents your graph, and methods in that class represents elements in that graph. 
```

Comment:
This is not really a change, is it? You added some new lines, but the mark down rendering is still the same, right?
If so, I suggest not to make a change here.